tools/llm_bench/benchmark.py

```diff
@@ -229,15 +230,17 @@ def get_argprser():
                         help="Path to .bin or .pt file with speaker embeddings for text to speech scenarios")
     parser.add_argument("--vocoder_path", type=str, default=None,
                         help="Path to vocoder  for text to speech scenarios")
+    parser.add_argument("-vf", "--video_frames", type=int, default=None,
+                        help="controller of video frames to process (required frame number or decymation factor if negative)")
```

Comment:
decym_frames = int(decym_frames)

```
    if decym_frames > 0:
        if len(output_frames) <= decym_frames:
            log.info(f"Video decym: too short to decym: crop: {decym_frames}")
            return list(output_frames[:decym_frames])
        decym_factor = 1 + int(len(output_frames) / decym_frames)
    else:
        decym_factor = -decym_frames
    log.info(f"Video decym factor: {decym_factor}")
    if decym_factor >= 2:
        return list(output_frames[::decym_factor])
    log.info("Video decym: too large decym factor: skip")
```

frames num = 100
1. decym_frames = 101 -> output_frames num will be 100, decym_frames is not work
2. decym_frames = 10 -> works as decym_frames, output_frames num will be 11
4. decym_frames = -1 -> works as decym_frames, output_frames num will be 100